packages/forms-web-app/src/views/eligibility/granted-or-refused-permission-out.njk

```diff
@@ -0,0 +1,18 @@
+{% extends "layouts/main.njk" %}
+
+{% set showBackLink = false %}
+{% block pageTitle %}
+  You need to use a different service - Eligibility - Appeal a householder planning decision - GOV.UK
+{% endblock %}
+
+{% block content %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">You need to use a different service</h1>
+      <p class="govuk-body">This is a new service. It is currently only available for appeals about refused planning permission.</p>
+      <p>You can <a data-cy="appeal-decision-service" href="https://acp.planninginspectorate.gov.uk/">appeal using our appeals casework portal</a>.</p>
```

Comment:
This paragraph element should have a class of `govuk-body` and the anchor should have a class of `govuk-link` to keep the typography consistent.